dial_test.go

```diff
@@ -161,8 +156,9 @@ func newSilentPeer(t *testing.T) (peer.ID, ma.Multiaddr, net.Listener) {
 func TestDialWait(t *testing.T) {
 	t.Parallel()
 
-	ctx := context.Background()
```

Comment:
again here, why are we deopping this context?

Reply:
Because it didn't do anything, it's a `context.Background()`.

Reply:
ok.